### DISCUSSION TITLE
Undo force close channel

### DISCUSSION THREAD

**Discussion by mcbglug** [lightningnetwork/lnd#8810] [Q&A]
Hi, i wrongly launched a force close channel, the transaction is in mempool now, is there a way to undo the force close? i can contact the guy on the other side of the channel.

Thanks

**Comment by ellemouton** [comment]
Unfortunately you cannot undo a force close transaction: Once a transaction is in your mempool, it may have been propagated to other mempools too meaning that your remote party may very well know of this transaction. If you two were to continue with normal channel operations & the channel was ever in a state where your side of the channel had more funds that it did compared to the state in the forceclose transaction, then nothing stops your peer from just broadcasting that old force-close transaction and essentially stealing funds from you.
